### ISSUE TITLE
csProj2ts Doesn't return the expected values

### ISSUE THREAD

**Issue by OClement** [TypeStrong/csproj2ts#7]
I have a .Net 4.5 project that I want to extract the Typescript configs from;
It was created with VS 2015 Update 1

When configuring the project through VS, I set ModuleKind to System, outDir to some directory under the project's folder, etc

However, when reading the csProj with csProj2ts, I don't get the values I set;

> [Here's the csproj](http://pastebin.com/pKHf9NZ4)
> [This is the output I get](http://pastebin.com/zDBjW7dL)  see inline comments

Not sure why there are properties that seems to be duplicates? (I realize there is a "default" typescript config, but where does it take this data from? In the csproj file source, those typescript configs are set only once)

Here's the call I make to generate the ouput above:
```
    tsConfig = csproj2ts.getTypeScriptSettings({
                ProjectFileName: csProj,
                ActiveConfiguration: 'Debug',
            }).then(function (res) {
                console.log(
                    '\n\n======\n',
                    res);
            });
```

Not sure what could cause this?

**Comment by nycdotnet** [comment]
Thanks for the report.  I haven't tested this with VS2015 yet but I didn't expect things would change much.  I will take a look.

**Comment by nycdotnet** [comment]
Thanks so much for the report.  This is really helpful.

I think that the issue is the conditions are different.  The conditions built-in to csproj2ts are fairly naïve at this time.

Do I have permission to include your provided csproj in this repo as a test artifact?

**Comment by nycdotnet** [comment]
OK I've got this fixed on my local machine.  Just pending your agreement to include the project as a test artifact.  Thanks so much for the report.

**Comment by OClement** [comment]
Hey
No problems 
If I recall correctly I stripped anything that could be considered sensitive information, so there shouldn't be any problems in adding it in there
If you happen to see something that might identify as "personal" information, just replace it with something else please

Thanks

**Comment by nycdotnet** [comment]
Thank you.

**Comment by nycdotnet** [comment]
This is released in v 0.0.7.  Thanks for the report and please let me know if you have any further issues.

Note there were a few things that aren't bugs - such as the version of TypeScript is something that you can set versus something that gets read out of the project.  I know this needs better docs, but it might only be relevant for a little while longer anyway with `tsconfig.json` being the way of the future.

Thanks.
